nodes/core/core/20-inject.html

```diff
@@ -26,6 +26,14 @@
         <input type="text" id="node-input-topic">
     </div>
 
+    <div class="form-row" id="node-once">
+        <label for="node-input-once">&nbsp;</label>
+        <input type="checkbox" id="node-input-once" style="display:inline-block;width:auto;vertical-align:top;">
```

Comment:
actually I think vertical-align:baseline will be better - and maybe width:15px lays out better.

Reply:
done